starter/source/model/sets/map_tables.F

```diff
@@ -238,8 +238,7 @@ SUBROUTINE CREATE_MAP_TABLES ( MAP_TABLES ,MODE  ,
           ALLOCATE(INDEX_SORT(2*NSUBMOD))
 
           DO I=1,NSUBMOD
-             OFFSET = LSUBMODEL(I)%OFF_SUBMOD - LSUBMODEL(I)%OFF_SUBMOD0
-             ISORT(I) = LSUBMODEL(I)%NOSUBMOD - OFFSET
+             ISORT(I) = LSUBMODEL(I)%NOSUBMOD
```

Comment:
Maybe @SebVilleneuve or @mircea-altair could review this change

Reply:
ok for me